web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -33,14 +33,14 @@ const settings = {
         value: 'binary',
       },
       {
-        label: 'CSV',
+        label: 'Delimited Text',
         value: 'csv',
       }
     ]
   },
   csvSeparator: {
-    label: 'CSV Separator',
-    description: 'The csv file separator. Defaults to ,',
+    label: 'Field Separator',
+    description: 'The delimited text file separator. Defaults to ,',
```

Comment:
@xnikhil08 @hmc-cs-wcao This is out of scope for this ticket, but I think the labels for comma, pipe, and semi-colon should read as Comma, Pipe, and Semicolon. This makes them consistent with "Tab" and now "Other", as opposed to some labels being a symbol and some being a word.

Reply:
I think the clearest representation of a delimiter character is the character itself. We put "Tab" because that character is whitespace. We could potentially distinguish the "Other" label using the following:

--Other--
_Other_

Reply:
If needed, we can investigate this elsewhere here:
https://project.marklogic.com/jira/browse/DHFPROD-2841

Reply:
Okay, was just curious.